### ISSUE TITLE
Fix failing build and update npm packages.

### ISSUE THREAD

**Issue by bhargav175** [istanbuljs/istanbuljs.github.io#25]

**Comment by bcoe** [comment]
@bhargav175 would love the help getting builds running more smoothly 👍 and appreciate any other contributions you would like to make to this site.

We worked with the creator of Phenomic @MoOx to build out this website; I'd like to keep using it, and I think we can eventually take advantage of more of its features.

**Comment by bhargav175** [comment]
I agree. I cloned the repo and tried working on phenomic. Spent close to 20 hours on it. But the phenomic has a few issues with webpack 2 as well as webpack 1 and they are currently open in phenomic. I managed to fix the dev environment, but not the production build. That said, I will be working on it for the next couple of days and figure out a solution. Will take @MoOx  and other phenomic dev help.
